src/datetime.rs

```diff
@@ -0,0 +1,341 @@
+//! Support datetimes and timedeltas
+//!
+//! [The corresponding section][datetime] of the NumPy documentation contains more information.
```

Comment:
Let's add some document that explains the relationship with datetime module in Python std lib.

Reply:
Done.